portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,7 @@
 
 const DELETE_ALL_COMMENTS_WARNING = 'Careful! Do you really want to delete all comments?';
 const DELETE_COMMENT_WARNING = 'Are you sure you want to delete this comment?';
+const DELETE_COMMENT_FAIL = 'You can\'t delete a comment that\'s not yours!';
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

If you'd like, you can use the backtick character \` to use what's called a Template Literal which is used for string formatting, but also would allow you to avoid escaping the single quotes.